File: 387_first_unique_character_in_a_string.py
class Solution:
    def firstUniqChar(self, s: str) -> int:
        
        count=[0]*26;
        
        for c in s:
            count[ord(c) - ord('a')]+=1
            
            
        for i in range(len(s)):
            if(count[ord(s[i])-ord('a')]==1):
                return i
            
        return -1
        

File: 387_first_unique_character_in_a_string.cpp
class Solution {
public:
    int firstUniqChar(string s) {
      int count[26]={0};
        for(char c:s){
            count[c-'a']+=1;
            // cout<<  count[c-'a']<< " "<< c-'a';
        }
        cout<<endl;
        
        for(int i=0;i<s.length();++i){
          //  cout<<count[s[i]-'a']<<" ";
            if(count[s[i]-'a']==1)
                
            {
                return i; 
               
            }  
            
        }
        return -1;
    }
};
